server/apiCalls/koppsApi.js

```diff
@@ -43,16 +39,13 @@ const combineStartEndDates = async (syllabusStartDates) => {
 
 const filteredKoppsData = async (courseCode, lang, testCourse = null) => {
   try {
-    const { course = {}, publicSyllabusVersions } = testCourse || (await rawKoppsCourseData(courseCode, lang))
-    const { credits = null, title = null } = course
+    const { publicSyllabusVersions } = testCourse || (await rawKoppsCourseData(courseCode, lang))
     const sortedSyllabusStartDates = await getListOfValidFromSyllabusTerms(publicSyllabusVersions)
     const syllabusPeriods = await combineStartEndDates(sortedSyllabusStartDates)
     return {
       courseCode: courseCode.toUpperCase(),
-      courseTitle: parseOrSetEmpty(title),
       sortedSyllabusStart: sortedSyllabusStartDates,
```

Comment:
I saw your note in confluence mapping page:

_"Notes: We probably can remove sortedSyllabusStart from webContext and use it only to create syllabusPeriods."_ 

I agree, and I think now is a good time to remove it.

Reply:
Sure I will take care of that one as well.